Products/ZenUI3/browser/resources/js/zenoss/zenoss.js

```diff
@@ -1373,7 +1368,7 @@ Ext.apply(Zenoss.date, {
 Zenoss.date.renderWithTimeZone = function (value, format) {
     if (Ext.isNumeric(value)) {
         if (!format) {
-            format = "YYYY-MM-DD hh:mm:ss a z";
+            format = Zenoss.USER_DATE_FORMAT + ' ' + Zenoss.USER_TIME_FORMAT + ' z';
```

Comment:
On my system, `format` will have the value
```
YY/MM/DD hh:mm:ss a z
```
I'm not sure the `moment` library would perform the desired conversion here.